windows/sound.h

```diff
@@ -24,36 +24,34 @@
 
 #pragma once
 
-#include <QMutex>
```

Comment:
I'd expect "standard" includes (`<...>`) to come before "local" includes (`"..."`).

Reply:
we have multiple options here:

- we could use `SortIncludes: false` so the linter does not reorder includes at all
- we could define rules for the order using `IncludeCategories` (see https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormatStyleOptions.html) - that's in the end a list of regular expressions that define the categories and their order

Reply:
> we could use SortIncludes: false so the linter does not reorder includes at all

If we don't find a way for configuring our style, this would be OK I think. As you point out further options, I think I'd prefer to give those a try first. If it is possible to come up with general rules such as what @pljones mentioned, this would be good. I'd be no fan of listing multiple or even all include files explicitly in such a config.

Reply:
I have added include categories like this so standard includes come first and Q(t) includes get their own group:
```
IncludeBlocks: Regroup
IncludeCategories:
  - Regex: '^<Q.*>$'
  - Regex: '^<.*>$'
  - Regex: '^".*"$'
 ```

https://github.com/jamulussoftware/jamulus/pull/1127/files#diff-1026e0038b722990204a42bed8a6f7c0ec2302aa79e3fad1959d62ba968edfa2R14-R18